raiden_contracts/deploy/__main__.py

```diff
@@ -1,45 +1,106 @@
 """
 A simple Python script to deploy compiled contracts.
 """
+import json
 import logging
 from logging import getLogger
-from getpass import getpass
 
 import click
-import json
-
-from raiden_contracts.contract_manager import ContractManager, CONTRACTS_SOURCE_DIRS
-from web3 import Web3, HTTPProvider
-from eth_utils import (
-    is_address,
-    to_checksum_address,
-)
-
-from raiden_contracts.utils.utils import (
-    check_succesful_tx,
-)
-from raiden_contracts.constants import (
-    CONTRACT_ENDPOINT_REGISTRY,
-    CONTRACT_TOKEN_NETWORK_REGISTRY,
-    CONTRACT_SECRET_REGISTRY,
-    CONTRACT_CUSTOM_TOKEN,
-    DEPLOY_SETTLE_TIMEOUT_MIN,
-    DEPLOY_SETTLE_TIMEOUT_MAX,
-)
+from eth_utils import denoms, encode_hex, is_address, to_checksum_address
+from raiden_contracts.constants import (CONTRACT_CUSTOM_TOKEN,
```

Comment:
Can you please indent those on the next line? like [this](https://github.com/raiden-network/raiden/blob/master/raiden/app.py#L11)

Reply:
aarrghh... isort is doing things behind my back.